src/main/java/cartPage/TotalsTable.java

```diff
@@ -0,0 +1,14 @@
+package cartPage;
+
+import org.openqa.selenium.By;
+
+public class TotalsTable {
+
+    public final String totalsTable = "//table[@id='totals_table']";
+
+    public By subTotalLocator = By.xpath(totalsTable + "//tbody/tr[1]/td[2]/span");
+    public By flatShippingRateLocator = By.xpath(totalsTable + "//tbody/tr[2]/td[2]/span");
+    public By totalLocator = By.xpath(totalsTable + "//tbody/tr[3]/td[2]/span");
+
```

Comment:
It will be good to have some validation methods here?:)

Reply:
Of course but it will be useful in the future, because it is not needed for our current epic. But I can add a few methods :)